mhs/common/mhs_common/workflow/asynchronous_express.py

```diff
@@ -143,15 +117,56 @@ def _record_outbound_audit_log(self, end_time, start_time, acknowledgment):
         await wdo.set_outbound_status(wd.MessageStatus.OUTBOUND_MESSAGE_PREPARED)
         return None, http_headers, message
 
+    def _parse_soap_error_response(self, response):
+        try:
+            parsed_body = ET.fromstring(response.body)
+
+            if EbxmlErrorEnvelope.is_ebxml_error(parsed_body):
+                _, parsed_response = ebxml_handler.handle_ebxml_error(response.code,
+                                                                      response.headers,
+                                                                      response.body)
+                logger.warning('0007', 'Received ebxml errors from Spine. {HTTPStatus} {Errors}',
+                               {'HTTPStatus': response.code, 'Errors': parsed_response})
+            elif SOAPFault.is_soap_fault(parsed_body):
+                _, parsed_response, _ = handle_soap_error(response.code,
+                                                          response.headers,
+                                                          response.body)
+                logger.warning('0008', 'Received soap errors from Spine. {HTTPStatus} {Errors}',
+                               {'HTTPStatus': response.code, 'Errors': parsed_response})
+            else:
+                logger.warning('0009', "Received an unexpected response from Spine",
+                               {'HTTPStatus': response.code})
+                parsed_response = "Didn't get expected response from Spine"
+        except ET.ParseError as pe:
+            logger.warning('0010', 'Unable to parse response from Spine. {Exception}', {'Exception': repr(pe)})
+            parsed_response = 'Unable to handle response returned from Spine'
+
+        return parsed_response
+
     @timing.time_function
     async def handle_inbound_message(self, message_id: str, correlation_id: str, work_description: wd.WorkDescription,
                                      payload: str):
+
+        logger.audit('0103', 'Async-Express inbound workflow invoked. Message received from spine')
+
         logger.info('0010', 'Entered async express workflow to handle inbound message')
         await wd.update_status_with_retries(work_description,
                                             work_description.set_inbound_status,
                                             wd.MessageStatus.INBOUND_RESPONSE_RECEIVED,
                                             self.store_retries)
 
+        await self._publish_message_to_inbound_queue(message_id, correlation_id, work_description, payload)
+
+        logger.info('0015', 'Placed message onto inbound queue successfully')
+        await work_description.set_inbound_status(wd.MessageStatus.INBOUND_RESPONSE_SUCCESSFULLY_PROCESSED)
+        logger.audit('0104', 'Async-Express inbound workflow completed. Message successfully processed, returning '
+                             '{acknowledgement}  to spine', {'acknowledgement': True})
```

Comment:
Can you do this as:
```suggestion
                             '{Acknowledgment}  to spine', {'Acknowledgment': True})
```
or the inverse. Just to match up the audit logs so they're a bit easier to search through.

Also, optional, should this perhaps be the message status instead of `True`, feels more consistent with the other log messages.